integration_tests/e2e/login.cy.ts

```diff
@@ -35,24 +35,14 @@ context('SignIn', () => {
     Page.verifyOnPage(AuthSignInPage)
   })
 
-  it('User can manage their details', () => {
-    cy.signIn()
-    const homePage = Page.verifyOnPage(HomePage)
-
-    homePage.manageDetails.get('a').invoke('removeAttr', 'target')
-    homePage.manageDetails.click()
-    Page.verifyOnPage(AuthManageDetailsPage)
-  })
-
   it('User sees links in footer but not Open Government Licence', () => {
     cy.signIn()
     Page.verifyOnPage(HomePage)
-
-    cy.get('.govuk-footer__inline-list .govuk-footer__link:visible').then(footerLinks => {
-      expect(footerLinks.length).equal(2)
-
-      const TsAndCsLink = footerLinks.last()
-      expect(TsAndCsLink).to.contain('Terms and conditions')
+    it('Fallback footer exists with no content', () => {
+      cy.signIn()
+      cy.task('stubDpsComponentsFail')
+      const indexPage = Page.verifyOnPage(HomePage)
+      indexPage.fallbackFooter.should('not.include.text', 'Feedback')
```

Comment:
the old footer and the current fallback footer don't contain 'Feedback' so what i think we can do here is `indexPage.fallbackFooter.should('include.text', 'Terms and conditions')` if you do put the `{{ govukFooter(…) }}` into footer.njk